mobile/package.json

```diff
@@ -41,7 +41,8 @@
         "react-native-splash-screen": "3.3.0",
         "react-native-toast-message": "2.1.6",
         "react-native-vector-icons": "10.0.0",
-        "react-redux": "8.1.2"
+        "react-redux": "8.1.2",
+        "toggle-switch-react-native": "3.3.0"
```

Comment:
https://reactnative.dev/docs/switch#value

Reply:
I knew about value. But then we shall change text style in this element too.... After thinking, I decide that it's better to use our component with our styles.
Then I shouldn't copy styles that we already have.
I was wrong? We should use label?

Reply:
I mean react native already have switch, we can skip additional library for it